common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OAuth2Strategy.java

```diff
@@ -81,7 +81,7 @@ public GenericAuthorizationResponse requestAuthorization(
         Uri authorizationUri = createAuthorizationUri(); //NOPMD Suppressing PMD warning for unused variable
         AuthorizationResult result = authorizationStrategy.requestAuthorization(request); //NOPMD Suppressing PMD warning for unused variable
         //TODO: Reconcile authorization result and response
-        AuthorizationResponse response = new AuthorizationResponse();
+        AuthorizationResponse response = new AuthorizationResponse("");
```

Comment:
We should have the AuthorizationResult consuming logic here. 
If AuthorizationResult has successful auth code returned,continue.
If failed, here AuthorizationErrorResponse returned, we throw the exception.

Reply:
You can make it a TODO and make a separate PR for this change.

Reply:
Sure. I wasn't aware about this. I will push another PR for this. There is a TODO already here.